Tasks/JenkinsDownloadArtifacts/jenkinsdownloadartifacts.ts

```diff
@@ -16,6 +16,10 @@ import { AzureStorageArtifactDownloader } from "./AzureStorageArtifacts/AzureSto
 import { ArtifactDetailsDownloader } from "./ArtifactDetails/ArtifactDetailsDownloader";
 import { JenkinsRestClient, JenkinsJobDetails } from "./ArtifactDetails/JenkinsRestClient"
 
+var packagejson = require('./package.json');
+
+const area: string = 'jenkinsdownloadartifacts';
```

Comment:
JenkinsDownloadArtifacts to be consistent with the other task?

Reply:
done